Casks/voov-meeting.rb

```diff
@@ -1,22 +1,28 @@
 cask "voov-meeting" do
-  version "1.7.1.510,1410000198"
-  sha256 "7e43ecc30d0eb99ab511bb8709b89d6c3916a5a05b638d1c4980210669ac669a"
+  version "3.3.2.510,1410000198"
+  sha256 :no_check
+
+  if Hardware::CPU.intel?
+    url "https://updatecdn.meeting.qq.com/255661c36b01c1f58129754358b192ac/VooVMeeting_#{version.csv.second}_#{version.csv.first}.publish.x86_64.dmg",
+        verified: "updatecdn.meeting.qq.com"
+  else
+    url "https://updatecdn.meeting.qq.com/4c95e5f9ac1a586edab01d761aa8e0be/VooVMeeting_#{version.csv.second}_#{version.csv.first}.publish.arm64.dmg",
+        verified: "updatecdn.meeting.qq.com"
+  end
 
```

Comment:
```suggestion
  arch = Hardware::CPU.intel? ? "x86_64" : "arm64"

  if Hardware::CPU.intel?
    version "3.3.2.510,1410000198,255661c36b01c1f58129754358b192ac"
    sha256 "e33c32b2af728c3ac0277156ef75e781fcec48f7191dc93c81669cde9b13dd64"

  else
    version "3.3.2.510,1410000198,4c95e5f9ac1a586edab01d761aa8e0be"
    sha256 "062e9ed7bf86dd64087100b595b3bd2e406ceb0aa5d1cf288513b05424bde783"
  end

  url "https://updatecdn.meeting.qq.com/#{version.csv.third}/VooVMeeting_#{version.csv.second}_#{version.csv.first}.publish.#{arch}.dmg",
      verified: "updatecdn.meeting.qq.com"
```

As was suggested above, I believe the hash will need to be part of the version, since it's part of the URL. This suggested change follows the pattern found in `dotnet` but the cask maintainers may suggest something different after review.